docs/application-connector/docs/010-details-ac-deployment.md

```diff
@@ -1,15 +1,17 @@
 ---
-title: Deploy a new Application Connector
+title: Deploy a new Remote Environment
 type: Details
 ---
 
-By default, Kyma comes with two Application Connectors preconfigured. Those Application Connectors are installed in the `kyma-integration` Namespace.
+By default, Kyma comes with two Remote Environments preinstalled. Those Remote Environments are installed in the `kyma-integration` Namespace.
 
-### Install an Application Connector locally
+>**NOTE:** A Remote Environment represents a single connected external solution.
 
-For installation on Minikube, provide the NodePort as shown in this example:
+## Install a Remote Environment on a local Kyma deployment
```

Comment:
Deployment?

Reply:
This is just a synonym for "installation", shouldn't be capitalized.